graph-db/indexer/handler.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports.index = async event => {
+  console.log(`Ryan, Sarah and Mark will see ingest event: [${JSON.stringify(event, null, 2)}].`)
```

Comment:
Should remove personal identifiers

Reply:
See my comment: `This function will be completely replaced with the actual indexing code that is coming when we work on CMR-7307. So ignore what's happening inside here.`